static/app/components/performanceOnboarding/step.tsx

```diff
@@ -0,0 +1,108 @@
+import 'prism-sentry/index.css';
+
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import CheckboxFancy from 'sentry/components/checkboxFancy/checkboxFancy';
+import space from 'sentry/styles/space';
+import {Project} from 'sentry/types';
+import localStorage from 'sentry/utils/localStorage';
+
+type Props = {
+  docContent: string | undefined;
+  docKey: string;
+  project: Project;
+};
+
+function OnBoardingStep(props: Props) {
+  const {docKey, project, docContent} = props;
+
+  const [increment, setIncrement] = useState<number>(0);
+
+  if (!docContent) {
+    return null;
+  }
+
+  const localStorageKey = `perf-onboarding-${project.id}-${docKey}`;
```

Comment:
I thought we're using clientState for this?

Reply:
@Neo-Zhixing I intentionally left out the clientState implementation as this PR has gotten fairly large. I kept it simple for now. 

It's definitely coming! 😄